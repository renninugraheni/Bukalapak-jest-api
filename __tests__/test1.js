const penjumlahan = require('../aritmatika');

describe('test jest', () => {
    test('testing to see if jest works', () => {
        expect(1).toEqual(1);
    });
    test('penjumlahan', () => {
        const hasil = penjumlahan(3,6);
        expect(hasil).toEqual(9);
        expect(hasil).toMatchSnapshot();
    });
}); 