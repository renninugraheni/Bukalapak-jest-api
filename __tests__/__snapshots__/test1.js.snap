// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test jest penjumlahan 1`] = `9`;
